hexa/core/schema.py

```diff
@@ -0,0 +1,63 @@
+import pathlib
+
+from ariadne import ObjectType, QueryType, load_schema_from_path
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import gettext_lazy
+
+from hexa.app import get_hexa_app_configs
+from hexa.catalog.models import Index
+from hexa.core.activities import Activity, ActivityList
+from hexa.core.models.behaviors import Status
+from hexa.plugins.connector_airflow.models import DAG
+from hexa.plugins.connector_s3.models import Object
+
+core_type_defs = load_schema_from_path(
+    f"{pathlib.Path(__file__).parent.resolve()}/graphql/schema.graphql"
+)
+
```

Comment:
You didn't create the `Activity` object type. I think you should created it

Reply:
Okay